big_tests/tests/mam_helper.erl

```diff
@@ -1105,8 +1105,8 @@ is_mam_possible(Host) ->
     is_cassandra_enabled(Host) orelse is_elasticsearch_enabled(Host).
 
 is_riak_enabled(_Host) ->
-    case rpc(mim(), mongoose_riak, get_worker, []) of
-        Pid when is_pid(Pid) ->
+    case rpc(mim(), mongoose_riak, list_buckets, [<<"default">>]) of
```

Comment:
Would be nice to have some kind of `get_status` function in `mongoose_wpool` for such checks. Can you either add it or leave a TODO behind?

Reply:
did you mean `mongoose_riak`?

Reply:
`mongoose_riak` as well. It would use `mongoose_wpool`'s `get_status`. :)